wire/message.go

```diff
@@ -22,9 +22,28 @@ const MessageHeaderSize = 24
 // header.  Shorter commands must be zero padded.
 const CommandSize = 12
 
-// MaxMessagePayload is the maximum bytes a message can be regardless of other
-// individual limits imposed by messages themselves.
-var MaxMessagePayload = uint32(1024*1024*32) * 2 // 32MB
+// ebs is the excessive block size, used to determine reasonable maximum message sizes.
+var ebs uint32
+
+// SetLimits initializes various message limits which change depending on the maximum
+// block size we will deal with.
+func SetLimits(excessiveBlockSize uint32) {
+	ebs = excessiveBlockSize
+}
+
+func getEBS() uint32 {
+	if ebs == 0 {
+		panic("message size limits have not been initialized")
```

Comment:
Set a reasonable default here and don't panic. Then you don't need to init the tests.

Reply:
If we do that, it is equivalent to the original usage of defaults which ignores actual EBS and we are in the same position as before. Any case where it is not set, the behavior becomes out of spec. I thought failing loudly and explicitly would be better than running outside spec?

Reply:
(it's a little better since all the values are actually dependent on the default instead of manually calculated across constants.)

Reply:
You already do a sanity check of the value when it is read from the config. Since we know we are going to set it to a reasonable value I have no problem with a default here. =)

Reply:
used and documented a default on `var ebs`